mono/mini/mini.h

```diff
@@ -1084,6 +1094,13 @@ typedef struct {
 	 * Stores if we need to run a chained exception in Windows.
 	 */
 	gboolean mono_win_chained_exception_needs_run;
+
+	/*
+	 * Thread-local state and time tracking.
+	 */
+	gint64 perf_segment_start;
+	gint64 perf_counters [MONO_PERF_STATE_NUM];
+	MonoThreadPerfState state;
```

Comment:
Do you think the new counter type will be used by code in metatada or utils?
If it's expected to happen soon, it might make sense to move them to utils now instead of later.